src/pmdas/prometheus/pmdaprometheus.1

```diff
@@ -0,0 +1,107 @@
+'\"macro stdmacro
+.\"
+.\" Copyright (c) 2017 Ronak Jain.
+.\"
+.\" This program is free software; you can redistribute it and/or modify it
+.\" under the terms of the GNU General Public License as published by the
+.\" Free Software Foundation; either version 2 of the License, or (at your
+.\" option) any later version.
+.\"
+.\" This program is distributed in the hope that it will be useful, but
+.\" WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+.\" or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+.\" for more details.
+.\"
+.\"
+.TH PMDAPROMETHEUS 1 "PCP" "Performance Co-Pilot"
+.SH NAME
+\f3pmdaprometheus\f1 \- Prometheus PMDA
+.SH DESCRIPTION
+\fBpmdaoprometheus\fR is a Performance Metrics Domain Agent (PMDA) which imports
+metrics from Prometheus endpoints which expose text based Prometheus metrics.
+.PP
+Further details on the Prometheus exposition format can be found at https://prometheus.io/docs/instrumenting/exposition_formats
+.SH METADATA SOURCES
+\fBpmdaprometheus\fR scans the $PCP_PMDAS_DIR/prometheus/metadata directory, looking for metadata files (*.json). These JSON metadata files describe the names, types, etc. of their associated Prometheus metrics for a given endpoint. For example, let's assume the following simple Prometheus metric:
```

Comment:
Paths are usually printed underlined in PCP man pages, so that would be:

.B pmdaprometheus
scans the
.I $PCP_PMDAS_DIR/prometheus/metadata
directory, looking for metadata files (\c
.IR *.json
).

Each line should be at most 80 characters (in the raw troff format), so I'd split the above line into multiple lines for text editor readability.